vspace/src/main/java/edu/asu/diging/vspace/web/publicview/ExhibitionSlideController.java

```diff
@@ -128,4 +131,17 @@ public String slide(Model model, @PathVariable("slideId") String slideId, @PathV
         model.addAttribute("spaceName", spaceManager.getSpace(spaceId).getName());
         return "exhibition/module";
     }
+
+    private void currentSlideHavingBranchingPoint(Model model, String slideId, boolean back, ISlide currentSlide) {
```

Comment:
besides that a slide is or is not a branching point (it doesn't have a branching point), why not just `handleBranchingPoint` or `processBrachingPoint`?